archive.conf

```diff
@@ -11,8 +11,8 @@ ARCHIVE_USER=archive
 ARCHIVE_GROUP=archive
 
 # Package extensions
-PKGEXT='.pkg.tar.xz'
-PKGSIG="$PKGEXT.sig"
+PKGREGEX='\.pkg\.tar(\.(gz|bz2|xz|zst|lrz|lzo|Z|lz4|lz))?'
+PKGSIG="$PKGREGEX\.sig"
```

Comment:
looks good, maybe change this to `PKGSIGREGEX` or such to be orthogonal to `PKGREGEX`